prepare/metrics/llm_as_judge/pairwise_rating/llama_3_ibm_genai_arena_hard_template.py

```diff
@@ -0,0 +1,38 @@
+from unitxt import add_to_catalog
+from unitxt.inference import (
+    IbmGenAiInferenceEngine,
+    IbmGenAiInferenceEngineParams,
+)
+from unitxt.llm_as_judge import LLMAsJudge
+
+model_list = ["meta-llama/llama-3-8b-instruct", "meta-llama/llama-3-70b-instruct"]
+format = "formats.llama3_chat"
+templates = [
+    "templates.response_assessment.pairwise_comparative_rating.arena_hard",
+    "templates.response_assessment.pairwise_comparative_rating.arena_hard_with_shuffling",
+]
+for template in templates:
+    task = "pairwise_comparative_rating.single_turn"
+
+    gen_params = IbmGenAiInferenceEngineParams(max_new_tokens=2048, random_seed=42)
+    for model_id in model_list:
+        inference_model = IbmGenAiInferenceEngine(
+            model_name=model_id, parameters=gen_params
```

Comment:
We noew have a new way to passing params (just by passing max_new_tokens=2048, random_seed=42).  No needed for parameter object. The old way is still supported, but is deprecated. We should change all internal code to use it.

Reply:
ok. Thou again, all these "smart" coding practice are totally unreadable.

Reply:
The parameters  are still there in a class:


class IbmGenAiInferenceEngineParamsMixin(Artifact):
    beam_width: Optional[int] = None
    decoding_method: Optional[Literal["greedy", "sample"]] = None
    include_stop_sequence: Optional[bool] = None
    length_penalty: Any = None
    max_new_tokens: Optional[int] = None
    min_new_tokens: Optional[int] = None
    random_seed: Optional[int] = None
    repetition_penalty: Optional[float] = None
    return_options: Any = None
    stop_sequences: Optional[List[str]] = None
    temperature: Optional[float] = None
    time_limit: Optional[int] = None
    top_k: Optional[int] = None
    top_p: Optional[float] = None
    truncate_input_tokens: Optional[int] = None
    typical_p: Optional[float] = None
    
     It's just that they are inherited, and not contained as a separate object.